Firmware/ultralcd.cpp

```diff
@@ -6326,7 +6333,11 @@ static void mmu_cut_filament_menu()
             bFilamentPreheatState=true;
             mFilamentItem(target_temperature[0],target_temperature_bed);
         }
-        else lcd_generic_preheat_menu();
+        else
+        {
+        	bPreheatOnlyNozzle = false;
```

Comment:
So this one area I have some doubts on. Shouldn't "cut filament" also be a good candidate for bPreheatOnlyNozzle = true?

Reply:
Since I've never seen this functionality working, I left it untouched. But I think you are right.